### DISCUSSION TITLE
Alternate close button behavior

### DISCUSSION THREAD

**Discussion by Martok88** [khmyznikov/pwa-install#56] [Ideas]
This is pretty much exactly what I was looking for. Thanks for developing and sharing it. 
I just have a minor issue with how closing the dialog is handled. Currently, the dialog will stay hidden for the active session, but I'd like to hide it for a set time period using a cookie or local storage. I've been trying to implement this behavior myself, but I can't figure out how to add an event listener to the close dialog button. 
A new event that fires on the user clicking the close button would be greatly appreciated.

**Comment by khmyznikov** [comment]
Hmm I think the `pwa-user-choice-result-event` is right what you're looking for? If user click close, you should see `dismissed` message.

**Reply by Martok88** [reply]
Ok, but it wouldn't work on iOS, right? Is there any way around that?

**Reply by khmyznikov** [reply]
It will work, you can try for yourself in demo

**Reply by Martok88** [reply]
Great. I was somewhat confused by this line in the README:

> ⚠️ success/fail/choice events is Chromium only, iOS don't have them.

**Reply by khmyznikov** [reply]
Yeah this is right, but "dismissed" is not in that list 🙂
